### ISSUE TITLE
Error on autocomplete

### ISSUE THREAD

**Issue by thomasballinger** [n-riesco/ijavascript#24]
I'm not sure I'm using the proper versions of things, but tab completion isn't working for me:
~~~
tomb@tom-mba ~$ node --version
v0.12.2
tomb@tom-mba ~$ ijs --version
3.0.0
tomb@tom-mba ~$ ipython --version
3.0.0
tomb@tom-mba ~$ ipython console --kernel js
[ZMQTerminalIPythonApp] CRITICAL | Could not find kernel js
tomb@tom-mba ~$ ipython console --kernel javascript
KERNEL: SHELL_SOCKET: Unhandled message type: history_request
IPython Console 3.0.0

In [1]: a = "asdf"
Out[1]: 'asdf'

In [2]: a.KERNEL: Exception in complete_request handler: TypeError: Cannot read property 'slice' of undefined
    at parseExpression (/usr/local/lib/node_modules/ijavascript/lib/handlers_v4.js:219:26)
    at Kernel.complete_request (/usr/local/lib/node_modules/ijavascript/lib/handlers_v4.js:339:22)
    at Kernel.onShellMessage (/usr/local/lib/node_modules/ijavascript/lib/kernel.js:177:41)
    at emit (events.js:118:17)
    at Socket._flush (/usr/local/lib/node_modules/ijavascript/node_modules/zmq/lib/index.js:543:17)
    at _zmq.onReady (/usr/local/lib/node_modules/ijavascript/node_modules/zmq/lib/index.js:197:12)

~~~

**Comment by n-riesco** [comment]
This bug is caused by IJavascript receiving IPython messages in the v5 protocol. I see several options how to proceed:
1. Implement v5 protocol (this is in my TODO list)
2. Patch TAB completion to accept both v4.1 and v5 messages (I'm concerned other messages would need patching)
3. Report this bug to the jupyter-console repository (I've already checked the bug is present in their latest commit; I've checked their protocol adapter doesn't get called)

At the moment, I think option 1 is the safest in terms of using my time effectively.
